src/swap/defi/thorchain.ts

```diff
@@ -319,23 +320,23 @@ export function makeThorchainPlugin(
         throw new SwapCurrencyError(swapInfo, fromCurrencyCode, toCurrencyCode)
       }
 
-      // Add outbound fee
-      const networkFee = calcNetworkFee(
-        toMainnetCode,
-        toCurrencyCode,
-        outboundGasRate,
-        pools
-      )
+      // // Add outbound fee
+      // const networkFee = calcNetworkFee(
+      //   toMainnetCode,
+      //   toCurrencyCode,
+      //   outboundGasRate,
+      //   pools
+      // )
 
       // Per Thorchain specs, user pays 2x the outbound transaction networkfee
-      let feeInDestCurrency = mul(networkFee, '2')
+      let feeInDestCurrency = outboundFee
```

Comment:
This appears to be missing the logic to convert the fee into the outbound asset for when the outbound asset is a token